nodejs/week2/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,68 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+
+
+router.get("/", async (request, response) => {
+
+  if(request.query.title){
+      const title = meals.filter((meal) =>
+      meal.title.toLowerCase().includes(request.query.title.toLowerCase()));
+      response.send(title);
+  }
+  else if(request.query.maxPrice){
+      
+    if(!parseInt(request.query.maxPrice)){
+         response.status(400)
+         response.send(`There is no meal with ${request.query.maxPrice} `);
```

Comment:
her you are checking if a number was send with the request? you're not looking into the data, as the response message suggests.